### ISSUE TITLE
Imperial Hoth does not give tokens

### ISSUE THREAD

**Issue by ChavexOrmand** [SWG-Source/dsrc#387]
As described, when completing imperial Hoth, no tokens or painting is rewarded.  Rebel side works fine and looks like its handled differently, as the imp side references imperial_quest_tracker, but in that it calls out Tusken King?

**Comment by Cekis** [comment]
Wait, what?  Tusken King?  Do you see that in the code?  Where?

**Comment by ChavexOrmand** [comment]
In sku.0/sys.server/compiled/game/script/theme_park/heroic/echo_base there is  an imperial_quest_tracker, which looks like it has nothing to do with anything hoth

**Comment by RDevReign** [comment]
@ChavexOrmand - The "imperial_quest_tracker" script is not referenced anywhere. It was used as a testing script. The "echo_quest_tracker" is the actual quest tracker that is used for both Imperial and Rebel Hoth. See screenshot below of it being referenced for both Imp/Reb Hoth.

![image](https://github.com/user-attachments/assets/0a0aae2d-1967-4e03-85f4-73513f564f52)

**Comment by ChavexOrmand** [comment]
Yeah I agree, did some further digging into it after I posted this. Regardless, the imp side of Hoth isn't granting the rewards in a similar fashion.  This was just the low hanging fruit I had noticed

**Comment by ChavexOrmand** [comment]
Its been fixed on numerous other projects, figured one of the smarter kids would take a look
